css-layout-classes/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>CSS Layout Classes</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+<div class="container">
+  <header >
+    <h1 class="full-column">Choose Your Character!</h1>
+  </header>
+  <div class="row">
+    <img src="images/link.png" alt="Link" class="column-one-third">
+    <div>
```

Comment:
you want to have a `column-one-third` class on this div so it is a proper column container